test/tinycore.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh -e
+#!/usr/bin/env sh
+set -e
```

Comment:
I don't see the point of `/usr/bin/env` here. Standards basically require /bin/sh to exist on any Unix-y system (unlike for other interpreters like bash or python where env might make sense).